### DISCUSSION TITLE
「編集する」をもっと素早く選択できるようにしてほしい

### DISCUSSION THREAD

**Discussion by aoyama-val** [increments/qiita-discussions#469] [要望・アイデア・不具合]
「…」でメニューを表示してから選択するのは面倒です。
- SNSのアイコンと同列に「編集する」のアイコンを設置する
- ショートカットキーを用意する

など、ワンアクションで実行できるようにしてほしいです。
![image](https://user-images.githubusercontent.com/13144822/234130570-db3596b5-6b65-4dcd-a88a-d611b6d80b04.png)

**Comment by tanaka-asami415** [comment]
@aoyama-val 
ご意見ありがとうございます🙏
同様のリクエストは他にもいただいているため、検討いたします！
